operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -16,7 +16,7 @@ powerscale:
       observability: "v1.6.0"
       resiliency: "v1.7.0"
   v2.9.0:
-      authorization: "v1.8.0"
+      authorization: "v1.9.0"
```

Comment:
I believe there are other instances where authorization should be incremented for a driver version in this file

Reply:
Those instances have been updated to reflect the appropriate driver version.